_src/pattern-library/sass/base/_reset.scss

```diff
@@ -104,6 +104,13 @@ pre {
 // ----------------------------
 .sr-only {
     @extend %a11y-hide;
+    background: black !important; // providing contrast
```

Comment:
@talbs Do you think using variables is a good idea? This isn't something that can be compromised and if the variables are changed or customized, it could lead to test failures. Granted, this is only used for automated testing - this isn't something that is seen visually. I'll use variables for now, but keep in mind that by using variables you don't necessarily guarantee passing tests.